sourcegraph/learn
# Add content: How to Find Error Messages with Sourcegraph (Video)

## Type of change

Add new content.

### What should this PR do?

Add a new piece of content: "How To Find Error Messages with Sourcegraph (Video)"

Ticket: [DEVED-90](https://sourcegraph.atlassian.net/browse/DEVED-90)

### What are the acceptance criteria? 

- The content page loads correctly
- The content card displays in the card lists correctly
- Video is embedded correctly and can be played
- All newly added links work correctly
- Front-matter fields are complete and are accurate
- Transcript is present

### How should this PR be tested?

Open the deploy preview and verify that the content renders correctly and meets the acceptance criteria.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
